installer/x86_64/platforms_asic

```diff
@@ -0,0 +1,3 @@
+x86_64-kvm_x86_64-r0
+x86_64-kvm_x86_64_4_asic-r0
+x86_64-kvm_x86_64_6_asic-r0
```

Comment:
Do we need to merge it? Or it will be auto-generated?

Reply:
No, this is not expected. Looks like an auto-generated file got added in.